virtual_rainforest/models/animals/model_schema.json

```diff
@@ -22,12 +22,27 @@
                             "diet": {
                                 "type": "string",
                                 "default": ""
+                            },
+                            "metabolic_type": {
+                                "type": "string",
+                                "default": ""
+                            },
+                            "birth_mass": {
+                                "type": "number",
+                                "default": 0.0
+                            },
+                            "adult_mass": {
+                                "type": "number",
+                                "default": 0.0
                             }
                         },
                         "required": [
                             "name",
                             "taxa",
```

Comment:
Pedantic point, but should this be "taxon"? :stuck_out_tongue: 

Obvs not important.

Reply:
haha, yes it should!

Reply:
I thought about this a little more and remembered why it's plural in the first place! Right now, I'm handing it a single level of classification. I'm expecting this category to expand a bit into multiple levels of classification and so left it with a plural. I'm going to leave it plural for now and we'll see what the final use of a taxa category looks like.